definitions/variable/technology/technologies.yaml

```diff
@@ -76,13 +76,45 @@
     unit: [EUR_2020/pkm/yr, USD_2010/pkm/yr]
     skip-region-aggregation: true
 - Variable Cost|Heat|Residential and Commercial|{Fuel}:
-    description: Capital cost for {Fuel}
+    description: Variable cost for {Fuel}
     unit: [EUR_2020/MWh, USD_2010/MWh]
     skip-region-aggregation: true
 - Variable Cost|Heat|Industrial|{Fuel}:
-    description: Capital cost for {Fuel}
+    description: Variable cost for {Fuel}
+    unit: [EUR_2020/MWh, USD_2010/MWh]
+    skip-region-aggregation: true
+- Variable Cost (incl. Fuel Cost)|Electricity|{Electricity Input}:
```

Comment:
I suggest to instead interpret "Variable Cost" as *only* the non-fuel-related costs and then add a separate variable "Fuel Cost|Electricity|..."

Reply:
Well, I am also going to add the variable 'Fuel Cost' but the way of getting the Variable Cost (incl. Fuel Cost) is not at all trivial and this is an output of Genesys-Mod so I would like to keep the Variable Cost (incl. Fuel Cost) as sth that can be retrieved from the platform as it is a required input for some models.
But we can remove the Variable Cost (excl. Fuel Cost) and add in the description of 'Variable Cost' that it is excluding the Fuel cost (which is not trivial and was previously mis-interpreted by more than one modelling team)

Reply:
I agree with Daniel (and this is also how we have the variable costs natively in our model). So in my understanding, the variable costs are just those associated with the technology itself.

I can provide them separately without an issue, so adding a separate variable sounds good to me!